apps/client/src/app/components/admin-market-data/asset-profile-dialog/asset-profile-dialog.component.ts

```diff
@@ -203,4 +207,31 @@ export class AssetProfileDialog implements OnDestroy, OnInit {
     this.unsubscribeSubject.next();
     this.unsubscribeSubject.complete();
   }
+
+  public importHistoricalData() {
```

Comment:
Can you move up? `public ngOnDestroy()` should always be the last (public) function...